src/components/PokedexGrid/PokedexGrid.css

```diff
@@ -0,0 +1,13 @@
+.all-pokemon {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+    overflow-x: hidden;
+    overflow-y: scroll;
+    text-align: center;
+    font-family: 'Ubuntu', sans-serif;
+    background: url("https://lh3.googleusercontent.com/ohXfK_gejGdGoxN7Vzf2hE9EWsvXhR9AnPlRqylsbhSFn1hU2pDZ4dkG8Q8Wj8cd4g9hVGShd2X7qBSTkbWiX-2UmhH44u_-M4xpO2nMBYsi0ymKeMpfZFOMWXtTJW28YdTjVquwquv9pTTCBFpTnG_C8euEUgO08mXO63oDcqIN9bmSvh5B8MUDw2MJ_oYAMJoCi8UeI9nvfs7z3UbcQQPNYHyXgi0wKW9aMmz8ogaRUPHf5Z7tKv9Am8my9XsleXNzeZjlTY7qobFffw5xwpNBUGNYdPWfkF-XWQyhrcJsESwGFHUlFDtVvU_V6IOgHA-jZmnmu8e3_wBA50VTO713U-DoDMjhZliK1qSUoR9FBxcPE7Z_O3kVk9v5qFUgwO2wmybVKDmH9gn55vDnBEiHCr-0cfKJgJHYlgIFm3fwguGYtIEpYK5ucGAxjz_NPcM5fw9BNQ-M67dyM5eY9Tdav198oMNTAJLKGmdaEG4v8UqRsxk5hyFhOWeufn6vuaG2TrxB9_kDR5gaYroneuDaYB9CySvKyXo_AmqA3po7mGl8W0HBphFHzOPkfFqyGdpTfeJNurtKuuV99cb57kHxwieMwWCPwUNRFzZwOByzNyfcPkeVMgy8z7tpYj6Eb9SUpt8Y-6j9-Qj9lfxnHJNTZ0CTzxvJ-N_Mwya-YA=w1135-h638-no") no-repeat;
+    background-size: cover;
+    height: 100vh;
+    margin: 0;
+    
+}
```

Comment:
Styling is looking great! Maybe we can think about changing the size of the grid and give it some sort of overflow scroll so that it doesn't take up a huge amount of screen

Reply:
What about removing names and borders, just some simply little icons for each one?

Reply:
@hayleyw7 
The good about the names, is if you are not sure about the Pokemon name, you probably know his icon, so you can also found it

Reply:
@Matt-Roden 
Thank you for the comment, The css already have a grid and has an overflow , please take a look into it and 🤗let's make it even better!